### ISSUE TITLE
Open a new pane by prompting the user for which profile to use

### ISSUE THREAD

**Issue by zadjii-msft** [microsoft/terminal#3586]
Bear with me for a second. Somebody suggested this in another issue, and I wonder if we should consider it here.

![image](https://user-images.githubusercontent.com/14316954/60481298-9fd73800-9c41-11e9-92a5-61b8f7e61ecf.png)

The new pane is focused, and put in a [number entry] or [search for profiles by name] mode, just like one of our earlier issues suggested.

We probably want that UI _anyway_, since it has some benefits. Thoughts?

_Originally posted by @DHowett-MSFT in https://github.com/microsoft/terminal/issues/1756#issuecomment-507507450_

**Comment by rkeithhill** [comment]
Yes, ^ this please!!  When I open a new pane it is often to ssh to another machine.  I have profiles that execute the ssh command with the username@machine name - basically, I just have to type in my password.  When I'm not able to use the profile, I have to type all that stuff in - blech.  :-)

**Comment by mdtauk** [comment]
Perhaps this could be the UI when there is no default profile set?

**Comment by Windos** [comment]
Love this idea, 100% support it!

**Comment by ADTC** [comment]
Copying here: My own idea from #10402 _New Tab page UI for choosing the profile to open from a grid list of profile icons_

## New Tab UI for choosing the profile to open

At this moment, opening Windows Terminal (without specifically choosing a profile) would open a window with the default profile. Some power-users such as myself may need to open a different profile based on our immediate need. This would require me to open a second tab with the correct profile, then close the tab with the default one. (Or we'd have to create profile shortcuts in our Desktop or Start Menu.)

Modern web browsers have a built-in New Tab page which often shows frequent shortcuts or recently visited pages to be helpful to the user in deciding where they would like to go. A simplified version of such a page could be added to Windows Terminal, that simply displays a list of all the profiles. Choosing one could open it in the new tab that's already created and waiting for a profile.

![image](https://user-images.githubusercontent.com/6047296/121598937-f4fa6600-ca74-11eb-835b-54648951d7df.png)

In Settings, the _Default Profile_ selection can have an option called _"Show the New Tab page."_ Consequently, the New Tab button on the tab bar would always show the New Tab page instead of opening a profile. This further increases the usefulness of this feature by giving the same convenient UI to pick a profile. (This may negate the need for the drop-down which could be removed or made optional.)

**Note:** The checkbox, if checked, would set the chosen profile to the _Default Profile_ selection in Settings. Subsequently, all new tabs and windows will open in that profile until the user returns to the Settings and resets the selection to _"Show the New Tab page."_ (The checkbox would be unchecked again.)

**Comment by JVimes** [comment]
I "split and ask" shortcut would be nice! I'm in the same boat as @rkeithhill.
